markets/perps-market/contracts/storage/PerpsAccount.sol

```diff
@@ -126,7 +126,7 @@ library PerpsAccount {
     ) internal view returns (uint256 availableWithdrawableCollateralUsd) {
         (
             bool isEligible,
-            uint256 availableMargin,
+            int256 availableMargin,
```

Comment:
can it be negative at some point?

Reply:
it can, yes.  account pnl could continue to get very negative, even past the liquidation margin until all your margin is gone + more.  hopefully liquidation bots liquidate the account before this happens, but still possible.

good test case actually for liquidation of an account with negative margin!